course_info/static/course_info/js/controllers/PeopleController.js

```diff
@@ -632,6 +632,10 @@
         $scope.selectedRole = $scope.roles[0];
         $scope.setCourseInstance($routeParams.courseInstanceId);
 
+        // configure the alert datatable
+        $scope.dtOptionsWarning = {searching: false, paging: false,
```

Comment:
might just be me, but once it spills over to multiple lines, i usually prefer one-line-per-key for dicts/objects:

```
$scope.dtOptionsWarning = {
    info: false,
    ordering: ([1, 'asc']),
    paging: false,
    searching: false,
};
```